### PR TITLE
HAFNIUM procedures

### PR DESCRIPTION
**Pull request by bfuzzy1**

### CODE REVIEW

File: discovery/26c8b8b5-7b5b-4de1-a128-7d37fb14f517.yml [hunk 1]
```diff
@@ -19,6 +12,17 @@ platforms:
     cmd:
       command: |
         nltest /dsgetdc:%USERDOMAIN%
+      variants:
+        - command: nltest /dclist:#{discovery.domain.name}
```
Comment (khyberspache) [inline discovery/26c8b8b5-7b5b-4de1-a128-7d37fb14f517.yml:16]: There is actually a domain fact! `#{domain.T1016}` for example

File: execution/1ccc757b-cd16-4f62-bf06-91019a9bed44.yml [hunk 2]
```diff
@@ -14,5 +14,5 @@ metadata:
   authors:
     - bfuzzy1
   tags:
-    - HAFNIUM
+    - hafnium
   version: 1
```
Comment (khyberspache) [inline execution/1ccc757b-cd16-4f62-bf06-91019a9bed44.yml:18]: increment version

**Review by khyberspache — CHANGES_REQUESTED** [review]
Solid change set

**Comment by bfuzzy1** [comment]
Changes made!

**Review by khyberspache — APPROVED** [review]

**Pull request merged**